github_api/voting.py

```diff
@@ -137,6 +138,9 @@ def get_vote_sum(api, votes):
     negative = 0
     for user, vote in votes.items():
         weight = get_vote_weight(api, user)
+        if random.randint(0,500) == 42:
+            #Jackpot!
+            weight = weight * 100
```

Comment:
`weight * 100` is a drastic increase - if this is going to be implemented, it should be much lower, like `* 1.1`, or `* 1.5` at the most.